### PR TITLE
CSS Selectors: add "Grouping Selectors"

### PR DESCRIPTION
**Pull request by ramiy**
Related:

https://github.com/mdn/sprints/issues/2233
https://github.com/mdn/sprints/issues/2164

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
The Travis run failed because you are trying to use a group type that doesn't exist. I'd suggest just using `Basic Selectors` instead of `Grouping Selectors`, as it is a pretty basic type.

**Comment by ramiy** [comment]
@chrisdavidmills 

How can I add a new group type for `Grouping Selectors`.

See related links:

https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors

https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/Universal_selectors
https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/Type_selectors

and the new grouping selectors:

https://wiki.developer.mozilla.org/en-US/docs/Web/CSS/Grouping_selectors

File: css/selectors.json [hunk 1]
```diff
@@ -44,6 +44,15 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Attribute_selectors"
   },
+  "Grouping selectors": {
```
Comment (ExE-Boss) [inline css/selectors.json]:
```suggestion
  "Selector list": {
```

File: css/selectors.json [hunk 2]
```diff
@@ -44,6 +44,15 @@
...
+    "syntax": ",",
+    "groups": [
+      "Grouping Selectors",
+      "Selectors"
+    ],
+    "status": "standard",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Grouping_selectors"
```
Comment (ExE-Boss) [inline css/selectors.json]:
```suggestion
    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Selector_list"
```

**Comment by chrisdavidmills** [comment]
Ah, thanks for sorting that out @ExE-Boss . You are right that we should use Selector list.

@ramiy to add new group type, you add it to the group list at

https://github.com/mdn/data/blob/e82e4caa73d88b0edcb52ae579c75e479c320997/css/definitions.json

File: css/selectors.json (hunk 1 above)
Comment (ramiy) [reply css/selectors.json]: Done.

File: css/selectors.json (hunk 2 above)
Comment (ramiy) [reply css/selectors.json]: Done.

**Comment by ExE-Boss** [comment]
Next, we’ll need [to update KumaScript’s `CSSRef` macro](https://github.com/mdn/kumascript/pull/1256) to account for the new sidebar group, like I did for combinators in mdn/kumascript#942.

**Comment by ExE-Boss** [comment]
A new version of [mdn‑data](https://www.npmjs.com/package/mdn-data) has to be published before this will show up on **MDN** (because of https://github.com/mdn/kumascript/pull/1126)

**Comment by ramiy** [comment]
The last version released [over 3 months ago](https://www.npmjs.com/package/mdn-data?activeTab=versions)... It's time to release a new version.

**Pull request merged**